test/export/test_export.py

```diff
@@ -4590,6 +4590,40 @@ def forward(self, x):
         }
         export(f, (inputs,), dynamic_shapes=dynamic_shapes)
 
+    def test_constant_aliasing(self):
```

Comment:
Can you do a rebase on https://github.com/pytorch/pytorch/pull/124978 and add three non-strict mode tests for script objects: 1. one as module attributes, 2. one as input and 3. when it's fakified as FakeScriptObject? Can use the TensorQueue or Foo. Examples can be found in test_torchbind.py.

Reply:
Will do, thanks